### PR TITLE
new UMT Hosted CE

### PR DESCRIPTION
**Pull request by jdost321**

### CODE REVIEW

File: topology/The University of Montana/UMontana IT/UMT-Hellgate.yaml [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+Production: true
+# SupportCenter is one of the support centers in topology/support-centers.yaml
+SupportCenter: Self Supported
+
+# GroupDescription is a long description of the resource group; may be multiple lines.
+GroupDescription: The University of Montana
+
+# If you have an up-to-date local git clone, fill GroupID with the output from `bin/next_resource_group_id`
+# Otherwise, leave it blank and we will fill in the appropriate value for you.
+GroupID: 1367
```
Comment (brianhlin) [inline topology/The University of Montana/UMontana IT/UMT-Hellgate.yaml:12]:
```suggestion
```

IDs are now automatically generated!

File: topology/The University of Montana/UMontana IT/SITE.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+AddressLine1: 32 Campus Drive
+Country: United States
+Description: The University of Montana
+ID: 10368
```
Comment (brianhlin) [inline topology/The University of Montana/UMontana IT/SITE.yaml:4]:
```suggestion
```

File: topology/The University of Montana/UMontana IT/UMT-Hellgate.yaml [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  # Resource Names need to be unique across all resources in the OSG.
+  UMT-Hellgate-CE1:
+    # Active is true if the resource is accepting requests, and false otherwise.
+    # When first registering a resource, set this to false. Set it to true when it's ready for production.
+    Active: false
+    # Description is a long description of the resource; may be multiple lines
+    Description: Hosted CE serving UMT
+    # If you have an up-to-date local git clone, fill ID with the output from `bin/next_resource_id`
+    # Otherwise, leave it blank and we will fill in the appropriate value for you.
+    ID: 1480
```
Comment (brianhlin) [inline topology/The University of Montana/UMontana IT/UMT-Hellgate.yaml:28]:
```suggestion
```

File: topology/The University of Montana/FACILITY.yaml [hunk 4]
```diff
@@ -0,0 +1 @@
+ID: 10213
```
Comment (brianhlin) [inline topology/The University of Montana/FACILITY.yaml:1]: I believe that this should be auto-generated as well. Can you try deleting this file?

**Review by brianhlin — APPROVED** [review]
LGTM. Will merge once comments are addressed

File: topology/The University of Montana/FACILITY.yaml (hunk 4 above)
Comment (jdost321) [reply topology/The University of Montana/FACILITY.yaml]: file removed

**Pull request merged**